test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -347,7 +347,7 @@ test_jinja_indent_templated_table_name_a:
             id
         FROM
             {{ product }}
-    {% if not loop.last -%} UNION ALL {%- endif %}
+        {% if not loop.last -%} UNION ALL {%- endif %}
```

Comment:
The linter now returns nicer results for this test now that we removed the obsolete code from `discard_unsafe_fixes()`. Previously, the rules were generating these results, but some of the changes were discarded for "safety reasons".